src/objects/PointsHelper.js

```diff
@@ -11,7 +11,7 @@ export class PointsHelper extends InstancedMesh {
 
 	constructor( count = 1000, points = [] ) {
 
-		super( new SphereBufferGeometry( 0.025 ), new MeshBasicMaterial(), count );
+		super( new BufferGeometry( 0.025 ), new MeshBasicMaterial(), count );
```

Comment:
`BufferGeometry` does not have attributes. 

Is this a type here and you wanted to use `SphereGeometry` instead?

Reply:
Thanks - was working on too many things last night 😅